.github/workflows/algorithm.yml

```diff
@@ -58,8 +58,8 @@ jobs:
       - name: Test
         run: docker compose exec ${{ matrix.service }} pytest
 
-  security:
-    name: Security
+  sast:
+    name: Static Code Anaylysis Security
```

Comment:
Not sure how I feel about this name. Thoughts?

Reply:
Maybe one of the following:
- Static Code Security
- Static Code Security Analysis

Or just
- Static Application Security Testing